autocorrelation.py

```diff
@@ -156,6 +172,8 @@ def autocorrelation(input_stream: Stream, atom_keys: Collection, distance: int =
             time_took, count, count/time_took
         ))
 
+    log.d("Stats: {}".format(autocorr_net.state("Statistics", "Nope")))
```

Comment:
Guessing the `"Nope"` is the default value, I'd make it clear by naming the parameter input.